spec/components/custom_wizard/mapper_spec.rb

```diff
@@ -250,4 +276,135 @@
       user: user1
     ).perform).to eq(false)
   end
+
+  context "Output Templating" do
```

Comment:
@fzngagan "Output Templating" doesn't need to be capitalized.

Reply:
fixed